Return-path: <hristo@venev.name>
Envelope-to: rene@exactcode.de
Delivery-date: Sun, 28 May 2023 16:03:27 +0000
Received: from a1-bg02.venev.name ([213.240.239.49])
	by exactco.de with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_2)
	(envelope-from <hristo@venev.name>)
	id 1q3Ire-0002wA-Jk
	for rene@exactcode.de; Sun, 28 May 2023 16:03:27 +0000
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed; d=venev.name;
	s=default; h=Content-Transfer-Encoding:Message-Id:Date:Subject:To:From:
	Content-Type:Reply-To:Sender; bh=+lqrHXwA1p92qST4iNmRjVoXTxoyj8vuOPGc0T4eIYQ=
	; b=oZAIDIWeL7p2nmM2D9PXpNQcE2glvxXpEQhxm7YRMKUEdhMwmq2H1OMr1NCx/EBL75mcZq6ZP
	FYn2QPBOmzgDE9jN6E57d41aHDs++38N4BsvOzcq6CTkiAqf4C6F92UbLhUV6VA8g1X4iCp5HJCm2
	kPb+l6CnlPcwOBg0eYF3HSR9DkCMv15rwaLBkG3mJ5Obo/Hl0PXkD7wVDTtmp+2qgn6LTyfEtSkcO
	1CTKwbf7XDb/lfrHQmJ/z9jJji8vc9I+bsBKjMahIfsXOIrM3uJx0gwKkUYH517VeIvaSKRX7vFoC
	bJiWiFc74wQs7xezkXnmgpd6G8AWPuXAJtskFKtbeo92zEKKPhsylhA7gAgpwqGvM1oln/8MVDli9
	ybma1ZEjZfLPMuBzlypT3HtJMV90+NnjzMtjsCMOZIHfpcrF4CjQv8gatdEq7OItuazm8RpOZkYWV
	VgSEQAlpFblDh5s7hs6H9ReXhhXemrWqaO3DQywFM5elTMeOAEF65Tl4FtM2pFsT2OgmcRZbqKRs7
	KuEAA5DFgCarEktlEruEQmegnOP8vztYFN9a8dajX3QSqnHUX6NnuU3xKzuA4JgW8p/SVn9aNHI6I
	FuQXXzuO0LFyd1B/B4HFxLu3+LQp93TqoysDAUXJLwRi7K7kKUv48UnNI47YyJNo3/BZnEQ=;
Received: from a1-bg02.venev.name ([213.240.239.49] helo=pmx1.venev.name)
	by a1-bg02.venev.name with esmtps
	id 1q3Ilx-0012Kk-18
	(TLS1.3:TLS_AES_256_GCM_SHA384:256)
	(envelope-from <hristo@venev.name>);
	Sun, 28 May 2023 15:57:34 +0000
Received: from venev.name ([213.240.239.49])
	by pmx1.venev.name with ESMTPSA
	id 6ODjBux5c2QBxgMAdB6GMg:T3
	(envelope-from <hristo@venev.name>); Sun, 28 May 2023 15:57:34 +0000
From: Hristo Venev <hristo@venev.name>
To: =?UTF-8?q?Ren=C3=A9=20Rebe?= <rene@exactcode.de>
Cc: Hristo Venev <hristo@venev.name>
Subject: [PATCH 2/2] fixup! hwmon/spd5118: Add support for thresholds, alarms, and chip info
Date: Sun, 28 May 2023 18:56:52 +0300
Message-Id: <20230528155651.247195-3-hristo@venev.name>
X-Mailer: git-send-email 2.40.1
In-Reply-To: <BCF33206-A17C-48F0-A45D-0870B094290A@exactcode.de>
References: <BCF33206-A17C-48F0-A45D-0870B094290A@exactcode.de>
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Spam-Score: 3.8 (+++)
X-Spam-Report: Spam detection software, running on the system "exactcode.de", has
 identified this incoming email as possible spam.  The original message
 has been attached to this so you can view it (if it isn't spam) or label
 similar future email.  If you have any questions, see
 the administrator of that system for details.
 
 Content preview:  Signed-off-by: Hristo Venev <hristo@venev.name> --- drivers/hwmon/spd5118.c
    | 2 +- 1 file changed, 1 insertion(+), 1 deletion(-) diff --git a/drivers/hwmon/spd5118.c
    b/drivers/hwmon/spd5118.c index 5a6c1a0f82a8..1e51ca9b8d33 100644 --- a/drivers/hwmon/spd5118.c
    +++ b/drivers/hwmon/spd5118.c @@ -83,7 +83,7 @@ static bool spd5118_vendor_valid(u16
    reg) u8 pfx = reg & 0xff; u8 id = reg >> 8; [...] 
 
 Content analysis details:   (3.8 points, 5.0 required)
 
  pts rule name              description
 ---- ---------------------- --------------------------------------------------
  2.8 RCVD_BAD_ID            RCVD_BAD_ID
  0.0 BAYES_50               BODY: Bayesian spam probability is 40 to 60%
                             [score: 0.4683]
  1.0 DNS_FROM_AHBL_RHSBL    RBL: Envelope sender listed in dnsbl.ahbl.org

Signed-off-by: Hristo Venev <hristo@venev.name>
---
 drivers/hwmon/spd5118.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hwmon/spd5118.c b/drivers/hwmon/spd5118.c
index 5a6c1a0f82a8..1e51ca9b8d33 100644
--- a/drivers/hwmon/spd5118.c
+++ b/drivers/hwmon/spd5118.c
@@ -83,7 +83,7 @@ static bool spd5118_vendor_valid(u16 reg)
 	u8 pfx = reg & 0xff;
 	u8 id = reg >> 8;
 
-	if (!__builtin_parity(pfx) || !__builtin_parity(id))
+	if (!parity8(pfx) || !parity8(id))
 		return false;
 	id &= 0x7f;
 	if (id == 0 || id == 0x7f)
-- 
2.40.1

